core/src/test/scala/unit/kafka/server/AutoTopicCreationManagerTest.scala

```diff
@@ -38,7 +38,8 @@ import org.apache.kafka.common.security.auth.{KafkaPrincipal, KafkaPrincipalSerd
 import org.apache.kafka.common.utils.{SecurityUtils, Utils}
 import org.apache.kafka.coordinator.transaction.TransactionLogConfigs
 import org.apache.kafka.coordinator.group.{GroupCoordinator, GroupCoordinatorConfig}
-import org.apache.kafka.server.config.ServerConfigs
+import org.apache.kafka.coordinator.share.ShareCoordinator
+import org.apache.kafka.server.config.{ServerConfigs, ShareCoordinatorConfig}
```

Comment:
I wonder whether `ShareCoordinatorConfig` should be in package `o.a.k.coordinator.share`.

Reply:
Yes, there are some other interfaces and classes which need to be moved. 
I will raise a separate PR for that to not pollute this one.